### ISSUE TITLE
Modularization support of native libs to get AAB suitable for "instant apps"

### ISSUE THREAD

**Issue by dumblob** [vlang/vab#145]
I wonder whether `vab` strives to support splitting the app into (potentially many) pieces of independent "bundle parts" (Android App Bundle "features" aka "modules") as described in https://medium.com/androiddevelopers/a-patchwork-plaid-monolith-to-modularized-app-60235d9f212e to achieve the "instant apps" functionality incl. so called "dynamic features" (i.e. basically views downloaded later on demand).

See https://www.youtube.com/watch?v=L9J2e5PYXNg for context.

**Comment by Larpon** [comment]
It does split for architecture. Not much else currently. V is mostly focused on monolithic builds - which doesn't fit well into this feature of theirs. At first glance it also looks like it will will need features only available via Java packages via e.g. maven or google which will require better integration with Java. It's possible until someone proves it's not I guess 🙂

**Comment by dumblob** [comment]
>At first glance it also looks like it will will need features only available via Java packages via e.g. maven or google which will require better integration with Java.

Yep, this is more or less what I meant. But if it's possible as you say, then let's keep this open as a feature request :wink:.
